lerna.json

```diff
@@ -8,6 +8,5 @@
       "conventionalCommits": true
     }
   },
-  "useWorkspaces": true,
```

Comment:
so lerna latest version doesn't need this `useWorkspaces` config ?

Reply:
Yes
https://github.com/lerna/lerna/blob/v8.1.8/CHANGELOG.md#700-2023-06-08

Reply:
> Workspaces are used by default and useWorkspaces has been removed